### DISCUSSION TITLE
NumTD to be automated in pdf

### DISCUSSION THREAD

**Discussion by tamihuber** [paulwinterbauer/NBIC#93] [Q&A] [answered]
PW, Any idea when this update will go thru? Where NumTD =null, and tanks showing US disch in bw history section.

**Comment by paulwinterbauer** [comment, marked as answer]
I'll try to get it in the next update.

As an aside, I would use email for questions of this nature (or as a comment within the actual issue page in github).  I think this discussion section should be about system documentation type of discussions (the kind of thing you might want to search again for later).  I know, a lot of platforms, but if we have defined roles for everything I think they can all be maximized.

**Reply by tamihuber** [reply]
sounds good. I am definitely a bit confused as to what should be in discussions.

**Reply by tamihuber** [reply]
PS - I thought you said this was something KK had to handle...not something an update would take care of. Mentioning in case you thought it was for something else.

**Reply by paulwinterbauer** [reply]
Two separate issues here.  One for Karl is just adding NumTD as a required field in the autoresponse.  Second would be a flag for NumTD = 0 when disch is reported in the tanks section (potentially also something Karl could include, but def something I could add to the post review response).  The second was what I was thinking of adding for the next update.

**Reply by tamihuber** [reply]
Cool. The other flag needed for the responder is when NumTD>0 but no US disch listed in the tank section.  2827234

**Reply by karlsi** [reply]
I suggest we ignore NumTD:  it was dropped from the revised 2016 form during the design phase, after discussion of how to address the shortcomings of the original form, and is not included in the form specification in the CFR.  See

 https://www.ecfr.gov/current/title-33/chapter-I/subchapter-O/part-151/subpart-D#151.2060

The field was only added back to the form after a state complained.  If some states want to respond to ships about NumTD that is their prerogative but NBIC should not spend time on it.
